portfolio_project1.html

```diff
@@ -0,0 +1,46 @@
+	<!--Html for the top navigation bar-->
+	<nav class='link_bar'>
+		<a class='top_link' href='portfolio_home.html'>Home</a>
+	</nav>
+	<!--Html for the title-->
+	<div class='center_bar'>
+		<h1>Project 1</h1>
+	</div>
+	<!--Html for section 1 with text on the left-->
+	<section class ='paragraph'>
+		<h3>Event 1</h3>
+		<!--Flexbox for the text and image, changes to column-reverse after a media queery-->
+		<div class='text_box left'>
+		<p>"On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble that are bound to ensue; and equal blame belongs to those who fail in their duty through weakness of will, which is the same as saying through shrinking from toil and pain. These cases are perfectly simple and easy to distinguish. In a free hour, when our power of choice is untrammelled and when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business it will frequently occur that pleasures have to be repudiated and annoyances accepted. The wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other greater pleasures, or else he endures pains to avoid worse pains."</p>
+		<img class='img' src='jake_face.jpg' alt='my face'/>
+		</div>
+	</section>
+	<!--Html for section 2 with text on the right-->
+	<section class ='paragraph'>
+		<h3>Event 1</h3>
+		<!--Flexbox for the text and image, changes to column after a media queery-->
+		<div class='text_box right'>
+			<img class='img' src='jake_face.jpg' alt='my face'/>
+			<p>"On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble that are bound to ensue; and equal blame belongs to those who fail in their duty through weakness of will, which is the same as saying through shrinking from toil and pain. These cases are perfectly simple and easy to distinguish. In a free hour, when our power of choice is untrammelled and when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business it will frequently occur that pleasures have to be repudiated and annoyances accepted. The wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other greater pleasures, or else he endures pains to avoid worse pains."</p>
+		</div>
+	</section>
+	<!--html for a center break section and final section-->
+	<div class='center_bar'>
+		<h2>Event 3</h2>
+		<p> short break text</p>
+		<nav class='paragrph_link'>
```

Comment:
Misspelled paragraph; uncertain if this is affecting your code in any way 🤷‍♀️

Reply:
it's not the tags have been removed